pkg/util/restconfig/restconfig.go

```diff
@@ -17,9 +17,12 @@ import (
 
 // RestConfig returns the Kubernetes *rest.Config for a kubeconfig
 func RestConfig(env env.Interface, oc *api.OpenShiftCluster) (*rest.Config, error) {
-	config, err := clientcmd.Load(oc.Properties.AdminKubeconfig)
+	config, err := clientcmd.Load(oc.Properties.AROServiceKubeconfig)
```

Comment:
please decide which kubeconfig to use based on whether oc.Properties.AROServiceKubeconfig != "", then use that, rather than doing a fallback if clientcmd.Load fails for any reason